README.md

```diff
@@ -1,9 +1,13 @@
 GLFW.jl  [![Build Status][status]][travis]
 =======
 
-[Julia][1] interface to [GLFW][2], a multi-platform library for creating windows with OpenGL contexts and managing input and events. GLFW has native support for Windows, OS X and many Unix-like systems using the X Window System, such as Linux and FreeBSD.
+[Julia][1] interface to [GLFW][2], a multi-platform library for creating windows
```

Comment:
Heh I probably did it as I was reading since I don't wrap text in my editor. It is a habit developed from trying to keep LaTeX diffs small. I can undo it if you want.